scripts/_utils/utils.py

```diff
@@ -129,8 +129,23 @@ def process_svg(svg_url) -> Tuple[bool, Union[str, None], bool]:
         return True, '/tmp/verified-svg.png', True
     else:
         return False, svg_url, False
-        
-        
+
+
+def remove_transparency(im, file_url) -> Tuple[bool, Union[str, None], bool]:
```

Comment:
What is `im`?  Either use unambiguous variable names or document these parameters in the docstring. Is it an Image object? a filepath? a url?  Does this method expect only png images?  if yes, then needs to be in docstring.  If not, then document how it handles other media types.

Reply:
it is in the docstring

> Removes transparent background from png to opt for a black background

Reply:
removed ambiguity per [2291e85](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/72/commits/2291e853d1c3983bd9aa1c3cdd4b67edbd2540d4)

Reply:
@punctuations I still don't know what datatype `image` parameter is.  There's no type hint.  docstring jsut says it's a png but png isn't a datatype, is is a string url? a string path? a File object? an PIL Image object?  Ditto the others.